library/network_state.py

```diff
@@ -22,15 +22,38 @@
 author: "Wen Liang (@liangwen12year)"
 """
 
+EXAMPLES = r"""
```

Comment:
To make the format consistent, should we also add the `options: desired_state:` for network_state ?

Reply:
> To make the format consistent, should we also add the `options: desired_state:` for network_state ?

It was already there - https://github.com/linux-system-roles/network/blob/1a689f7eedbdd9b1adbd93552700bb817aa419df/library/network_state.py#L17

One thing to note is that this internal module documentation is intended for end users to call these modules directly, without using the role.  This use case does not work and is unsupported.  For us, this internal role documentation and formatting is really only so that we can pass ansible-lint and ansible-test gating tests.  For us, the main documentation is the README.md for the role, which describes the working and supported interface.  Users are not supposed to know or care that there are modules used internally in the role to implement the functionality of the role.